src/Types.hs

```diff
@@ -473,6 +473,7 @@ data AppState = AppState
     , _asError     :: Maybe Error -- ^ in case of errors, show this error message
     , _asViews     :: ViewSettings -- ^ stores widget and focus information
     , _asFileBrowser :: FileBrowser
+    , _asLocalTime :: UTCTime
```

Comment:
Is it possible to avoid putting this in the state at all, and ask the current time in the drawing function?

Reply:
We can't do IO in the drawing code, but we could in the event handling. Hmmm I think keeping it as a state variable is the only choice here...

Reply:
I think you're right.